corehq/apps/registration/views.py

```diff
@@ -128,28 +128,40 @@ def _create_new_account(self, reg_form):
             web_user.phone_numbers.append(reg_form.cleaned_data['phone_number'])
             web_user.save()
 
-        email = new_user.email
-
-        # registration analytics
-        persona = reg_form.cleaned_data['persona']
-        persona_other = reg_form.cleaned_data['persona_other']
-        appcues_ab_test = toggles.APPCUES_AB_TEST.enabled(web_user.username,
-                                                          toggles.NAMESPACE_USER)
-
-        track_workflow(email, "Requested New Account")
-        track_workflow(email, "Persona Field Filled Out", {
-            'personachoice': persona,
-            'personaother': persona_other,
-        })
+        if settings.IS_SAAS_ENVIRONMENT:
+            email = new_user.email
+            
+            # registration analytics
+            # only do anything with this in a SAAS environment
+
+            persona = reg_form.cleaned_data['persona']
+            persona_other = reg_form.cleaned_data['persona_other']
+            appcues_ab_test = toggles.APPCUES_AB_TEST.enabled(web_user.username,
+                                                              toggles.NAMESPACE_USER)
 
-        track_web_user_registration_hubspot.delay(
-            web_user,
-            {
-                'buyer_persona': persona,
-                'buyer_persona_other': persona_other,
-                "appcues_test": "On" if appcues_ab_test else "Off",
-            }
-        )
+            track_workflow(email, "Requested New Account", {
+                'environment': settings.SERVER_ENVIRONMENT,
```

Comment:
Curious why we're sending this - don't analytics already know what server they're on?

Reply:
i'm trying to gather whatever data is possible right now to identify if there is any bad data coming in